api/tests/Api/CampCollaborations/ResendInvitationCampCollaborationTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Tests\Api\CampCollaborations;
+
+use App\Entity\CampCollaboration;
+use App\Tests\Api\ECampApiTestCase;
+
+/**
+ * @internal
+ */
+class ResendInvitationCampCollaborationTest extends ECampApiTestCase {
+    // TODO security tests not collaborator
```

Comment:
Die ergänze ich dann in #1906 noch wenn du zuerst mergest

Reply:
Ich habe die security tests hinzugefügt.